### PR TITLE
feat(console): express integration guide

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add express integration guide in console.

## Testing

截屏2022-08-23 下午4 44 37

截屏2022-08-23 下午4 52 13

### CODE REVIEW

File: packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx [hunk 1]
```diff
@@ -3,374 +3,211 @@ import Step from '@mdx/components/Step';
 import Tabs from '@mdx/components/Tabs';
```
Comment (gao-sun) [inline packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx:3]: integration guide

Comment (wangsijie) [reply packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx:3]: updated

File: packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx [hunk 2]
```diff
@@ -3,374 +3,211 @@ import Step from '@mdx/components/Step';
...
 import TabItem from '@mdx/components/TabItem';
 import Alert from '@/components/Alert';
+import { generateRandomString } from '@logto/shared';
 
 <Step
-  title="Add Logto SDK as a dependency"
+  title="Add Logto SDK and other dependencies"
```
Comment (charIeszhao) [inline packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx:9]: nit: maybe just `Add dependencies`?

Comment (wangsijie) [reply packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx]: updated

File: packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx [hunk 3]
```diff
@@ -3,374 +3,211 @@ import Step from '@mdx/components/Step';
...
+app.use(session({ secret: '${generateRandomString(32)}', cookie: { maxAge: 14 * 24 * 60 * 60 } }));`}
 </code>
 </pre>
 
 </Step>
 
 <Step
-  title="Sign in"
-  subtitle="2 steps"
+  title="Sign In"
```
Comment (charIeszhao) [inline packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx:93]:
```suggestion
  title="Sign in"
```

File: packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx [hunk 4]
```diff
@@ -3,374 +3,211 @@ import Step from '@mdx/components/Step';
...
+  res.json(req.user);
 });
 ```
 
 </Step>
 
 <Step
-  title="Access protected resource"
-  subtitle="3 steps"
+  title="Protect Routes"
```
Comment (charIeszhao) [inline packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx:164]:
```suggestion
  title="Protect routes"
```

File: packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx [hunk 5]
```diff
@@ -3,374 +3,211 @@ import Step from '@mdx/components/Step';
...
+  res.end('protected resource');
 });
 ```
 
-```pug
-// views/index.jade
+</Step>
+
+<Step
+  title="Sign Out"
```
Comment (charIeszhao) [inline packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx:194]:
```suggestion
  title="Sign out"
```

**Review by charIeszhao — APPROVED** [review]
Use sentence case in card titles

**Comment by charIeszhao** [comment]
Shoot, I just found we were actually mix-using "Title Case" and "Sentense case" in previous guides... Should align

**Pull request merged**